src/commands/install-docker-credential-helper.yml

```diff
@@ -0,0 +1,84 @@
+description: >
+  Install a credential helper for Docker, automatically chosen
+  based on platform detection.
+  See: https://docs.docker.com/engine/reference/commandline/login/#credentials-store#credential-helpers
+  Supported platforms: Ubuntu/Debian and macOS.
+parameters:
```

Comment:
This could also be given a parameter for the helper the same as above. This would potentially reduce confusion in the future if we explicitly pass the values. Or both values could be inferred.